Typo Fix

IonicaBizau opened the pull request:
[`Matrici` -> `Matrice`](http://dexonline.ro/definitie/matrice)

Sper că am fixat peste tot:

```sh
$ grep -ri 'Matrici' .
./.git/COMMIT_EDITMSG:Fixed typo ("Matrice" instead of "Matrici").
./.git/logs/HEAD:ba0339aa00b07a593a78e778fdb33a7108d72bda 463476c9484a0553fe65333fed0859b7667f080d Ionică  Bizău <bizauionica@yahoo.com> 1418221156 +0200	commit: Renamed files ("Matrici" -> "Matrice").
./.git/logs/HEAD:463476c9484a0553fe65333fed0859b7667f080d 536a1211ed1ebb606540ac6459eca9c5cac7e5f6 Ionică  Bizău <bizauionica@yahoo.com> 1418221325 +0200	commit: Fixed typo ("Matrice" instead of "Matrici").
./.git/logs/refs/heads/typo:ba0339aa00b07a593a78e778fdb33a7108d72bda 463476c9484a0553fe65333fed0859b7667f080d Ionică  Bizău <bizauionica@yahoo.com> 1418221156 +0200	commit: Renamed files ("Matrici" -> "Matrice").
./.git/logs/refs/heads/typo:463476c9484a0553fe65333fed0859b7667f080d 536a1211ed1ebb606540ac6459eca9c5cac7e5f6 Ionică  Bizău <bizauionica@yahoo.com> 1418221325 +0200	commit: Fixed typo ("Matrice" instead of "Matrici").
$ grep --exclude-dir=".git" -ri 'Matrici' .
```

IonicaBizau commented:
/cc @HoreaOros

HoreaOros commented:
De ce nu era bun numele ```Matrici```?

IonicaBizau commented:
Trebuia să fac mai clar faptul că **nu** am transformat din plural în singular. :smile: [Pluralul este *matrice*, nu *matrici*](http://dexonline.ro/definitie/matrice).

HoreaOros commented:
Va trebui sa facem la Academia Romana o propunere de modificare a DEX-ului. Termenul 'matrici' este folosit foarte des de matematicieni. Mai devreme sau mai tarziu vor accepta si aceasta varianta de plural. 
Pana atunci incerc sa imi dau seama de ce nu pot face Merge automat.

IonicaBizau commented:
:smile: 

> Pana atunci incerc sa imi dau seama de ce nu pot face Merge automat.

Sunt conflicte între fișiere, probabil fișiere care sunt modificate și în `HoreaOros:master` și în `IonicaBizau:typo`. De aceea `git` nu poate să facă merge automat.

https://help.github.com/articles/resolving-a-merge-conflict-from-the-command-line/

O soluție rapidă ar fi să rezolv eu conflictele făcând un merge `HoreaOros:master` :arrow_right: `IonicaBizau:typo`, rezolvând conflictele, iar apoi acest PR va putea fi `merge`-uit automat.

IonicaBizau commented:
Acum se poate face merge automat. Problema era [aceasta](http://stackoverflow.com/q/5009096/1420197) (ceva diferențe între Windows și Unix/Linux).

Mai jos este modul cum am rezolvat problema:

```sh
$ git status
On branch master
Your branch is up-to-date with 'origin/master'.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   Cifre/Cifre.sln
        modified:   Cifre/Cifre/App.config
        modified:   Cifre/Cifre/Cifre.csproj
        modified:   Cifre/Cifre/Program.cs
        modified:   Cifre/Cifre/Properties/AssemblyInfo.cs
        modified:   Hello World/Hello World.sln
        modified:   Hello World/Hello World/App.config
        modified:   Hello World/Hello World/Hello World.csproj
        modified:   Hello World/Hello World/Program.cs
        modified:   Hello World/Hello World/Properties/AssemblyInfo.cs
        modified:   Suma2/Suma2.sln
        modified:   Suma2/Suma2/App.config
        modified:   Suma2/Suma2/Program.cs
        modified:   Suma2/Suma2/Properties/AssemblyInfo.cs
        modified:   Suma2/Suma2/Suma2.csproj
        modified:   TreiNPlus1/TreiNPlus1.sln
        modified:   TreiNPlus1/TreiNPlus1/App.config
        modified:   TreiNPlus1/TreiNPlus1/Program.cs
        modified:   TreiNPlus1/TreiNPlus1/Properties/AssemblyInfo.cs
        modified:   TreiNPlus1/TreiNPlus1/TreiNPlus1.csproj
        modified:   ec2grd/ec2grd.sln
        modified:   ec2grd/ec2grd/App.config
        modified:   ec2grd/ec2grd/Program.cs
        modified:   ec2grd/ec2grd/Properties/AssemblyInfo.cs
        modified:   ec2grd/ec2grd/Secvente.cs
        modified:   ec2grd/ec2grd/ec2grd.csproj
        modified:   ec2grd/ec2grd/ec2grd.csproj.user

no changes added to commit (use "git add" and/or "git commit -a")
ionicabizau@notebook:~/Documents/FP2014$ vim .gitattributes
ionicabizau@notebook:~/Documents/FP2014$ vim .gitattributes
ionicabizau@notebook:~/Documents/FP2014$ git status
On branch master
Your branch is up-to-date with 'origin/master'.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   .gitattributes

no changes added to commit (use "git add" and/or "git commit -a")
ionicabizau@notebook:~/Documents/FP2014$ git diff
diff --git a/.gitattributes b/.gitattributes
index 1ff0c42..38e9ecb 100644
--- a/.gitattributes
+++ b/.gitattributes
@@ -1,7 +1,8 @@
 ###############################################################################
 # Set default behavior to automatically normalize line endings.
 ###############################################################################
-* text=auto
+#* text=auto
+

 ###############################################################################
 # Set default behavior for command prompt diff.
@@ -17,7 +18,7 @@
 #
 # Merging from the command prompt will add diff markers to the files if there
 # are conflicts (Merging from VS is not affected by the settings below, in VS
-# the diff markers are never inserted). Diff markers may cause the following
+# the diff markers are never inserted). Diff markers may cause the following
 # file extensions to fail to load in VS. An alternative would be to treat
 # these files as binary and thus will always conflict and require user
 # intervention with every merge. To do so, just uncomment the entries below
@@ -46,9 +47,9 @@

 ###############################################################################
 # diff behavior for common document formats
-#
+#
 # Convert binary document formats to text before diffing them. This feature
-# is only available from the command line. Turn it on by uncommenting the
+# is only available from the command line. Turn it on by uncommenting the
 # entries below.
 ###############################################################################
 #*.doc   diff=astextplain
ionicabizau@notebook:~/Documents/FP2014$ l
Arrays/  Cifre/  ec2grd/  Hello World/  Matrici/  Operators/  README.md  Siruri/  Suma2/  Tablouri/  TreiNPlus1/
ionicabizau@notebook:~/Documents/FP2014$ git status
On branch master
Your branch is up-to-date with 'origin/master'.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   .gitattributes

no changes added to commit (use "git add" and/or "git commit -a")
ionicabizau@notebook:~/Documents/FP2014$ git checkout typo
M       .gitattributes
Branch typo set up to track remote branch typo from origin.
Switched to a new branch 'typo'
ionicabizau@notebook:~/Documents/FP2014$ git status
On branch typo
Your branch is up-to-date with 'origin/typo'.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   .gitattributes

no changes added to commit (use "git add" and/or "git commit -a")
ionicabizau@notebook:~/Documents/FP2014$ ls
Arrays  Cifre  ec2grd  Hello World  Matrice  Operators  README.md  Siruri  Suma2  Tablouri  TreiNPlus1
ionicabizau@notebook:~/Documents/FP2014$ git diff
diff --git a/.gitattributes b/.gitattributes
index 1ff0c42..38e9ecb 100644
--- a/.gitattributes
+++ b/.gitattributes
@@ -1,7 +1,8 @@
 ###############################################################################
 # Set default behavior to automatically normalize line endings.
 ###############################################################################
-* text=auto
+#* text=auto
+

 ###############################################################################
 # Set default behavior for command prompt diff.
@@ -17,7 +18,7 @@
 #
 # Merging from the command prompt will add diff markers to the files if there
 # are conflicts (Merging from VS is not affected by the settings below, in VS
-# the diff markers are never inserted). Diff markers may cause the following
+# the diff markers are never inserted). Diff markers may cause the following
 # file extensions to fail to load in VS. An alternative would be to treat
 # these files as binary and thus will always conflict and require user
 # intervention with every merge. To do so, just uncomment the entries below
@@ -46,9 +47,9 @@

 ###############################################################################
 # diff behavior for common document formats
-#
+#
 # Convert binary document formats to text before diffing them. This feature
-# is only available from the command line. Turn it on by uncommenting the
+# is only available from the command line. Turn it on by uncommenting the
 # entries below.
 ###############################################################################
 #*.doc   diff=astextplain
ionicabizau@notebook:~/Documents/FP2014$ git commit -m 'Commented out the text=auto line.' .
[typo d0d9998] Commented out the text=auto line.
 1 file changed, 5 insertions(+), 4 deletions(-)
ionicabizau@notebook:~/Documents/FP2014$ ls
Arrays  Cifre  ec2grd  Hello World  Matrice  Operators  README.md  Siruri  Suma2  Tablouri  TreiNPlus1
ionicabizau@notebook:~/Documents/FP2014$ git status
On branch typo
Your branch is ahead of 'origin/typo' by 1 commit.
  (use "git push" to publish your local commits)
nothing to commit, working directory clean
ionicabizau@notebook:~/Documents/FP2014$ ls
Arrays  Cifre  ec2grd  Hello World  Matrice  Operators  README.md  Siruri  Suma2  Tablouri  TreiNPlus1
ionicabizau@notebook:~/Documents/FP2014$ git pull git@github.com:HoreaOros/FP2014.git
remote: Counting objects: 56, done.
remote: Compressing objects: 100% (34/34), done.
remote: Total 56 (delta 22), reused 37 (delta 17)
Unpacking objects: 100% (56/56), done.
From github.com:HoreaOros/FP2014
 * branch            HEAD       -> FETCH_HEAD
Auto-merging README.md
CONFLICT (content): Merge conflict in README.md
Auto-merging .gitignore
CONFLICT (content): Merge conflict in .gitignore
Automatic merge failed; fix conflicts and then commit the result.
ionicabizau@notebook:~/Documents/FP2014$ vim README.md
ionicabizau@notebook:~/Documents/FP2014$ atom README.md
ionicabizau@notebook:~/Documents/FP2014$ git status
On branch typo
Your branch is ahead of 'origin/typo' by 1 commit.
  (use "git push" to publish your local commits)
You have unmerged paths.
  (fix conflicts and run "git commit")

Changes to be committed:

        new file:   Curs 16 Dec/Curs 16 Dec.sln
        new file:   Curs 16 Dec/Curs 16 Dec/App.config
        new file:   Curs 16 Dec/Curs 16 Dec/Curs 16 Dec.csproj
        new file:   Curs 16 Dec/Curs 16 Dec/Program.cs
        new file:   Curs 16 Dec/Curs 16 Dec/Properties/AssemblyInfo.cs
        new file:   Curs 9 dec/Curs 9 dec.sln
        new file:   Curs 9 dec/Curs 9 dec/App.config
        new file:   Curs 9 dec/Curs 9 dec/Curs 9 dec.csproj
        new file:   Curs 9 dec/Curs 9 dec/Program.cs
        new file:   Curs 9 dec/Curs 9 dec/Properties/AssemblyInfo.cs
        new file:   NewProject/NewProject.sln
        new file:   NewProject/NewProject/App.config
        new file:   NewProject/NewProject/NewProject.csproj
        new file:   NewProject/NewProject/Program.cs
        new file:   NewProject/NewProject/Properties/AssemblyInfo.cs
        new file:   Structuri/Structuri.sln
        new file:   Structuri/Structuri/App.config
        new file:   Structuri/Structuri/Program.cs
        new file:   Structuri/Structuri/Properties/AssemblyInfo.cs
        new file:   Structuri/Structuri/Structuri.csproj
        modified:   TreiNPlus1/TreiNPlus1/Program.cs

Unmerged paths:
  (use "git add <file>..." to mark resolution)

        both modified:   .gitignore
        both modified:   README.md

ionicabizau@notebook:~/Documents/FP2014$ vim .gitignore
ionicabizau@notebook:~/Documents/FP2014$ git c^C
ionicabizau@notebook:~/Documents/FP2014$ vim .gitignore
ionicabizau@notebook:~/Documents/FP2014$ git status
On branch typo
Your branch is ahead of 'origin/typo' by 1 commit.
  (use "git push" to publish your local commits)
You have unmerged paths.
  (fix conflicts and run "git commit")

Changes to be committed:

        new file:   Curs 16 Dec/Curs 16 Dec.sln
        new file:   Curs 16 Dec/Curs 16 Dec/App.config
        new file:   Curs 16 Dec/Curs 16 Dec/Curs 16 Dec.csproj
        new file:   Curs 16 Dec/Curs 16 Dec/Program.cs
        new file:   Curs 16 Dec/Curs 16 Dec/Properties/AssemblyInfo.cs
        new file:   Curs 9 dec/Curs 9 dec.sln
        new file:   Curs 9 dec/Curs 9 dec/App.config
        new file:   Curs 9 dec/Curs 9 dec/Curs 9 dec.csproj
        new file:   Curs 9 dec/Curs 9 dec/Program.cs
        new file:   Curs 9 dec/Curs 9 dec/Properties/AssemblyInfo.cs
        new file:   NewProject/NewProject.sln
        new file:   NewProject/NewProject/App.config
        new file:   NewProject/NewProject/NewProject.csproj
        new file:   NewProject/NewProject/Program.cs
        new file:   NewProject/NewProject/Properties/AssemblyInfo.cs
        new file:   Structuri/Structuri.sln
        new file:   Structuri/Structuri/App.config
        new file:   Structuri/Structuri/Program.cs
        new file:   Structuri/Structuri/Properties/AssemblyInfo.cs
        new file:   Structuri/Structuri/Structuri.csproj
        modified:   TreiNPlus1/TreiNPlus1/Program.cs

Unmerged paths:
  (use "git add <file>..." to mark resolution)

        both modified:   .gitignore
        both modified:   README.md

ionicabizau@notebook:~/Documents/FP2014$ git ^C
ionicabizau@notebook:~/Documents/FP2014$ ^C
ionicabizau@notebook:~/Documents/FP2014$ grep -ri 'Matrici' .
Binary file ./.git/index matches
ionicabizau@notebook:~/Documents/FP2014$ l
Arrays/  Cifre/  Curs 16 Dec/  Curs 9 dec/  ec2grd/  Hello World/  Matrice/  NewProject/  Operators/  README.md  Siruri/  Structuri/  Suma2/  Tablouri/  TreiNPlus1/
ionicabizau@notebook:~/Documents/FP2014$ git status
On branch typo
Your branch is ahead of 'origin/typo' by 1 commit.
  (use "git push" to publish your local commits)
You have unmerged paths.
  (fix conflicts and run "git commit")

Changes to be committed:

        new file:   Curs 16 Dec/Curs 16 Dec.sln
        new file:   Curs 16 Dec/Curs 16 Dec/App.config
        new file:   Curs 16 Dec/Curs 16 Dec/Curs 16 Dec.csproj
        new file:   Curs 16 Dec/Curs 16 Dec/Program.cs
        new file:   Curs 16 Dec/Curs 16 Dec/Properties/AssemblyInfo.cs
        new file:   Curs 9 dec/Curs 9 dec.sln
        new file:   Curs 9 dec/Curs 9 dec/App.config
        new file:   Curs 9 dec/Curs 9 dec/Curs 9 dec.csproj
        new file:   Curs 9 dec/Curs 9 dec/Program.cs
        new file:   Curs 9 dec/Curs 9 dec/Properties/AssemblyInfo.cs
        new file:   NewProject/NewProject.sln
        new file:   NewProject/NewProject/App.config
        new file:   NewProject/NewProject/NewProject.csproj
        new file:   NewProject/NewProject/Program.cs
        new file:   NewProject/NewProject/Properties/AssemblyInfo.cs
        new file:   Structuri/Structuri.sln
        new file:   Structuri/Structuri/App.config
        new file:   Structuri/Structuri/Program.cs
        new file:   Structuri/Structuri/Properties/AssemblyInfo.cs
        new file:   Structuri/Structuri/Structuri.csproj
        modified:   TreiNPlus1/TreiNPlus1/Program.cs

Unmerged paths:
  (use "git add <file>..." to mark resolution)

        both modified:   .gitignore
        both modified:   README.md

ionicabizau@notebook:~/Documents/FP2014$ git add .
ionicabizau@notebook:~/Documents/FP2014$ git commiit
git: 'commiit' is not a git command. See 'git --help'.

Did you mean this?
        commit
ionicabizau@notebook:~/Documents/FP2014$ git commit
[typo 1b3fa52] Merge github.com:HoreaOros/FP2014 into typo
ionicabizau@notebook:~/Documents/FP2014$ git push
Counting objects: 64, done.
Delta compression using up to 8 threads.
Compressing objects: 100% (59/59), done.
Writing objects: 100% (64/64), 9.97 KiB | 0 bytes/s, done.
Total 64 (delta 32), reused 1 (delta 0)
To git@github.com:IonicaBizau/FP2014.git
   536a121..1b3fa52  typo -> typo
ionicabizau@notebook:~/Documents/FP2014
```

IonicaBizau commented:
:fireworks:  :boom:  Landed in master! :fire: :star: :smile:

GhitaB commented:
> Va trebui sa facem la Academia Romana o propunere de modificare a DEX-ului. Termenul 'matrici' este folosit foarte des de matematicieni.

:+1:
